src/components/Vendors/VendorForm.jsx

```diff
@@ -73,9 +62,9 @@ export default function VendorForm({ ref, vendor, setFormData, setRefresh }) {
     }
   };
 
-  const awsFunctionCall = (func, vendor) => {
+  const awsFunctionCall = (func, ...vendorData) => {
```

Comment:
Using spread operator to enable variable number of parameter for differing `updateVendor` and `addVendor` function signatures to handle both with the same processing function.